dev/sg/main.go

```diff
@@ -532,6 +544,24 @@ func printRunUsage(c *ffcli.Command) string {
 	return out.String()
 }
 
+func migrationFixupExec(ctx context.Context, args []string) (err error) {
+	if len(args) != 0 {
+		out.WriteLine(output.Linef("", output.StyleWarning, "ERROR: too many arguments\n"))
+		return flag.ErrHelp
+	}
+
+	var (
+		databaseName = *migrationSquashDatabaseNameFlag
+		database, ok = databaseByName(databaseName)
+	)
```

Comment:
```suggestion
	databaseName := *migrationSquashDatabaseNameFlag
	
	database, ok := databaseByName(databaseName)
```

A bit odd that database/ok are defined and checked immediately, but the indentation doesn't match.